docs/source/keylime/index.rst

```diff
@@ -0,0 +1,97 @@
+Keylime
+=======
+
+ESI provides a guide to use `keylime`_ outside Ironic for run-time attestation with `IMA`_.
+
+Prerequisites
+-------------
+* A Keylime server machine with keylime registrar and verifier installed.
+* A node to be attested with an active TPM 2.0 and keylime agent installed.
+* A private network for attestation workflow.
```

Comment:
It feels like the prerequisites should be the 'private attestation network', and then simply one node that will act as the Keylime server.

Reply:
got it!